Jenkinsfile

```diff
@@ -112,6 +114,12 @@ pipeline {
                 steps: this
                 )
               def mapPatallelTasks = pythonTasksGen.generateParallelTests()
+
+              // Let's now enable the windows stages
+              readYaml(file: '.ci/.jenkins_windows.yml')['windows'].each { v ->
+                def description = "${v.VERSION}-${v.DISTUTILS_USE_SDK.equals('1') ? 'use-sdk' : ''}-${v.WEBFRAMEWORK}"
```

Comment:
This will help to run the same version with different parameters, for such the primary key is a combination of `VERSION`, `DISTUTILS_USE_SDK` and `WEBFRAMEWORK`.